### DISCUSSION TITLE
Gitlens update moves all views into 'Gitlens Inspect' view

### DISCUSSION THREAD

**Discussion by ldelossa** [gitkraken/vscode-gitlens#2811] [Q&A]
Hey,

I move most of my often used Gitlens views into the default source control view that VSCode provides. 

Every time I update Gitlens however it moves all the views back into a 'Gitlens Inspect' view and adds this view to the activity panel. 

Can I instruct Gitlens not to do this?

**Comment by eamodio** [comment]
@ldelossa This happened more than once? I should have only happened once for a few of the views (Commit Details & File/Line History) as they had their "home" moved, and because apparently VS Code doesn't save their position if you never moved those views at all. But that should have only happened once for that 14.0 release.

Can you add more details as to when you see it? And if you can a video of it would be helpful.

**Reply by ldelossa** [reply]
So far, it seems to happen on every update of either VSCode or Gitlens, not too sure which right now, since I updated both at the same time. 

I can't do a video, unless you push a test release/bump to gitlens, in which I can update it and reload and see if it occurs. But I've noticed this more then once now.

**Reply by eamodio** [reply]
See if it happens again on the upgrade to GitLens 14.1.1 which just went out.

**Reply by ldelossa** [reply]
No issue this time. Tho now I'm wondering if the issue actually occurs when VSCode updates on their monthly releases. I'll report back on the next monthly vscode release
